change/@fluentui-react-file-type-icons-0c5f2387-6307-4cee-9d80-1d1d56ad97b3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33455&commitId=4c51940bda10a1a67e95b439925e5598124ffe2e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.No beak.chromium.png|2306| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
visual regressions that showed up consisted of layout shifts on elements unrelated to the stuff changed in the PR